daffodil-japi/src/main/scala/org/apache/daffodil/japi/Daffodil.scala

```diff
@@ -835,6 +845,18 @@ class InvalidParserException private[japi] (cause: org.apache.daffodil.compiler.
  */
 class InvalidUsageException private[japi] (cause: org.apache.daffodil.processors.InvalidUsageException) extends Exception(cause.getMessage(), cause.getCause())
 
+/**
+ * This exception will be thrown when when the unparseResult.isError during a SAX Unparse
+ */
+class DaffodilUnparseErrorSAXException(exception: SDaffodilUnparseErrorSAXException)
+  extends org.xml.sax.SAXException(exception.getMessage)
+
+/**
+ * This exception will be thrown when an unexpected error occurs during the SAX unparse
```

Comment:
Might be worth documentation that if this happens there is no UnparseResult and getUnparseResult from teh content handler may return null, or something to that effect.